picard/ui/metadatabox.py

```diff
@@ -624,6 +624,9 @@ def _update_tags(self, new_selection=True, drop_album_caches=False):
 
             tag_diff.add('~length', str(orig_metadata.length), str(new_metadata.length),
                          removable=False, readonly=True)
+            if (len(files) == 1):
```

Comment:
This code snippet looks good to me. However you can combine the two if statements as follows:
 ```python
 if (len(files) == 1) and (settings['rename_files'] or settings['move_files']):
```

Reply:
Ok 👍